packages/medusa/src/migrations/1631104895519-RankColumnWithDefaultValue.ts

```diff
@@ -0,0 +1,23 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
Please update the filename to be snake_cased like all the other migrations -> https://github.com/medusajs/medusa/tree/master/packages/medusa/src/migrations